include/envoy/http/async_client.h

```diff
@@ -199,6 +210,11 @@ class AsyncClient {
       StreamOptions::setSendXff(v);
       return *this;
     }
+    RequestOptions& setHashPolicy(
+        const Protobuf::RepeatedPtrField<envoy::api::v2::route::RouteAction::HashPolicy>& v) {
+      hash_policy = v;
```

Comment:
```suggestion
      StreamOptions::setHashPolicy(v);
```

Sorry about the confusion above. I meant to tag this line.

Reply:
Ah, got it. Yes, updated, thanks.